src/xo-app/home/index.js

```diff
@@ -376,19 +380,29 @@ export default class Home extends Component {
                     </Popover>
                   }
                 >
-                  <Button className='btn-link'><span><Icon icon='pool' /> {_('homeAllTags')} ({tags.length})</span></Button>
+                  <Button className='btn-link'><span><Icon icon='tags' /> {_('homeAllTags')} ({tags.length})</span></Button>
                 </OverlayTrigger>
                ) : null
               }
               &nbsp;
-              <button className='btn btn-link dropdown-toggle'>
-                <Icon icon='filters' /> {_('homeSort')}
-              </button>
-              &nbsp;
-              <button className='btn btn-secondary'
-                onClick={this._expandAll}>
-                <Icon icon='nav' />
-              </button>
+              <DropdownButton bsStyle='link' id='sort' title='SortBy'>
```

Comment:
What is this `title` prop? If it's the displayed text, it should be translated.

Why is there an `id`? I don't think it's necessary.